beetmoverscript/task.py

```diff
@@ -17,6 +19,19 @@ def validate_task_schema(context):
     scriptworker.client.validate_json_schema(context.task, task_schema)
 
 
+def validate_task_scopes(context):
+    scopes = context.task['scopes']
+    for scope in scopes:
```

Comment:
This tests whether the scope exists.  I would like us to also only allow pushing to a nightly bucket / use nightly creds if we are using `project:releng:beetmover:nightly`, and same for release bucket / release creds if we're using `project:releng:beetmover:release`.